Libs/MRML/DisplayableManager/vtkMRMLViewInteractorStyle.h

```diff
@@ -35,14 +35,14 @@ class vtkTimerLog;
 ///
 /// Events are converted to new-style VTK interaction events and
 /// forwarded to displayable managers for processing.
-/// Some additional high-level events (cush as click and double-click)
+/// Some additional high-level events (such as click and double-click)
 /// are generated here.
 class VTK_MRML_DISPLAYABLEMANAGER_EXPORT vtkMRMLViewInteractorStyle :
-  public vtkInteractorStyleUser
+  public vtkInteractorStyle3D
 {
 public:
   static vtkMRMLViewInteractorStyle *New();
-  vtkTypeMacro(vtkMRMLViewInteractorStyle,vtkInteractorStyleUser);
+  vtkTypeMacro(vtkMRMLViewInteractorStyle,vtkInteractorStyle3D);
```

Comment:
This may not have been necessary. Now that I checked again, the OnMove3D and OnButton3D functions are available in vtkInteractorStyle, which is the subclass of vtkInteractorStyleUser (I think this is what I missed). Do you want me to change back?

Reply:
It is better to use as simple base class as possible, so if vtkInteractorStyle3D is not needed then change back to the old one.

Reply:
I know know... It was a subclass of the 3D interactor so that vtkVirtualRealityViewInteractorStyle can call Dolly3D and other functions that it needs. I'll revert OK?

Reply:
Such methods are moved to the camera widget. Handling such events n the he widget allows remapping of GUI events to various interactions and a more stable routing of events (e.g., the effect of a gesture may depend in what widget currently has the focus and what state it is in).

For short term you can choose any option, but in the mid/long term we are moving away from one centralized interactor style + some widgets and instead we will handle all interactions in widgets.

If the new model works well then we'll work on pushing it into VTK, thereby reducing redundancy.